sllurp/cli.py

```diff
@@ -169,3 +170,15 @@ def reset(host, port):
     Args = namedtuple('Args', ['host', 'port'])
     args = Args(host=host, port=port)
     _reset.main(args)
+
+
+@cli.command()
+@click.argument('host', type=str, nargs=-1)
+@click.option('-p', '--port', type=int, default=5084)
+def capabilities(host, port):
+    Args = namedtuple('Args', ['host', 'port'])
+    args = Args(host=host, port=port)
+    _capabilities.main(args)
+
```

Comment:
Could you remove these trailing blank lines?

Reply:
I just pushed a new commit for that.